src/index.ts

```diff
@@ -15,6 +15,7 @@ OPTIONS
   -o, --plugin-options PLUGIN=OPTS  JSON-encoded OPTS for PLUGIN (allows multiple).
   -r, --require PATH                Require PATH before transform (allows multiple).
       --extensions EXTS             Comma-separated extensions to process (default: "${Array.from(DEFAULT_EXTENSIONS).join(',')}").
+      --use-local-babel             Run plugins through babel plugins/presets specified in local .babelrc file instead of babel-preset-env
```

Comment:
nit: end this statement in a period.

Reply:
Also, should we add a `--no-transpile-plugins` option? Using `babel-register` does slow things down a bit and it might be nice to allow opting out of that. What do you think?

Reply:
is this something we can have as a separate, dependent issue on this feature? Just to keep this PR focused?